lib/src/note/note.dart

```diff
@@ -65,17 +55,35 @@ final class Note implements Comparable<Note>, Transposable<Note> {
   /// Example:
   /// ```dart
   /// Note.c.difference(Note.d) == 2
-  /// Note.eFlat.difference(Note.bFlat) == 7
+  /// Note.e.flat.difference(Note.b.flat) == 7
   /// Note.a.difference(Note.g) == -2
   /// ```
   int difference(Note other) => other.semitones - semitones;
 
+  /// Returns this [Note] sharpened by 1 semitone.
+  ///
+  /// Example:
+  /// ```dart
+  /// Note.c.sharp == const Note(BaseNote.c, Accidental.sharp)
+  /// Note.a.sharp == const Note(BaseNote.a, Accidental.sharp)
+  /// ```
+  Note get sharp => Note(baseNote, Accidental(accidental.semitones + 1));
```

Comment:
Perhaps you could add a helper on `Accidental` to increment the no. of semitones, although there is already `Accidental.increment` which has a different meaning.

Reply:
Yeah, and I’ve thought if the current `increment` meaning is worth its place here or if we can remove it in favor of the more convenient one. 🤔

Reply:
#260 renames the former method to `Accidental.incrementBy` and implements addition and subtraction operators.